lib/triggers/steamrepTrigger.js

```diff
@@ -37,11 +37,11 @@ SteamrepTrigger.prototype._respond = function(toId, message) {
 	if (steamid) {
 		var that = this;
 		var steamrep={};
-		that.winston.info("Checking steamrep.com for " + steamid);
+		that.winston.info(that.chatBot.name+"/"+that.name+": Checking steamrep.com for " + steamid);
 		request.get({method:"GET",encoding:"utf8",uri:"http://steamrep.com/api/beta/reputation/"+steamid+"?json=1&extended=1",json:true,followAllRedirects:true}, function(error, response, body) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/node-steam-chat-bot/steam-chat-bot/file/969592599/issues/source?bid=1980036&fileBranchId=2848073#l41)